test/integration/amphtml/test/index.test.js

```diff
@@ -246,6 +246,13 @@ describe('AMP Usage', () => {
       expect(await browser.elementByCss('#only-amp').text()).toMatch(/Only AMP/)
     })
 
+    it('should add data-ampdevmode to development script tags', async () => {
```

Comment:
We need to test that these are absent in production.

Reply:
The validator fails if they are present so our production tests that check it's valid check this